### PR TITLE
Add Together AI Demo

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

Adds a Together AI Template

## Test Plan

Tested it works on Appwrite Cloud

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: node/generate-with-together-ai/src/main.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+  if (req.method === 'GET') {
+    return res.send(getStaticFile('index.html'), 200, {
+      'Content-Type': 'text/html; charset=utf-8',
+    });
+  }
+
+  const models = {
+    'text': 'mistralai/Mixtral-8x7B-Instruct-v0.1',
+    'image': 'stabilityai/stable-diffusion-xl-base-1.0'
```
Comment (christyjacob4) [inline node/generate-with-together-ai/src/main.js:21]: does the API only support text and images? The README.md says audio as well on Line 3 and 21

File: node/generate-with-together-ai/static/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Replicate Demo</title>
```
Comment (christyjacob4) [inline node/generate-with-together-ai/static/index.html:8]:
```suggestion
  <title>Together AI Demo</title>
```
Please run a search through the entire folder to make sure we dont make this mistake in other places

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by PineappleIOnic** [comment]
@christyjacob4 Resolved, ready for re-review

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**